README.md

```diff
@@ -312,11 +312,26 @@ Like stubbing this call also uses the same DSL to filter and query requests.
 
 ### Scenario
 
-**Title** : Clean all scenarios status.<br>
+**Title** : Clean all scenarios status and pause state.<br>
```

Comment:
Did you mean unpaused in this case?

Reply:
I can see the ambiguity, I mean the state of the pause variable/switch. To me it reads as clearing all pause/status state back to default. Should I adjust?

Reply:
Gotcha! It's ok.